Foundation/Decimal.swift

```diff
@@ -661,17 +661,19 @@ extension Decimal {
     public mutating func formTruncatingRemainder(dividingBy other: Decimal) { fatalError("Decimal does not yet fully adopt FloatingPoint") }
 }
 
-extension Decimal: CustomStringConvertible {
+extension Decimal : CustomStringConvertible {
```

Comment:
Are we consistent in the spacing in the `extension` declarations? It seemed to be a mix, not sure if you just want to move everything to be one or other.

Reply:
Yes, I'm moving the formatting to be the same in the portion of the file that's common between sclf and the overlay.

I'm going to do (have already done) a gardening pass for formatting on the remainder of this file that is unique to sclf, but I left it out of this PR to keep things focused here on synchronizing the two files.